packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -243,7 +243,7 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
       classNames.main,
       {
         backgroundColor: palette.white,
-        boxShadow: '0px 0px 30px 0px rgba(0,0,0,0.2)',
+        boxShadow: '0px 25.6px 57.6px 0px rgba(0, 0, 0, 0.22), 0px 4.8px 14.4px 0px rgba(0, 0, 0, 0.18)',
```

Comment:
Same: these are fluent shadows that are not supposed to be in the default styles for v6.0.